.taskcluster.yml

```diff
@@ -0,0 +1,687 @@
+allowPullRequests: collaborators
```

Comment:
This configuration file is huge and fairly repetitive. I assume it has to be checked in to the repo like .travis.yml, but could parts of it be generated?

Reply:
Ha, there is a script :) Suggestion is then to add a comment at the top pointing to the script.